installing/installing_ibm_z/installing-ibm-z.adoc

```diff
@@ -54,7 +54,7 @@ include::modules/csr-management.adoc[leveloffset=+2]
 [id="additional-resources_ibmz-requirements"]
 .Additional resources
 
-* See link:https://www.ibm.com/docs/en/zvm/7.1?topic=networks-bridging-hipersockets-lan-zvm-virtual-switch[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in IBM Documentation.
+* See link:https://www.ibm.com/docs/en/zvm/latest?topic=networks-bridging-hipersockets-lan-zvm-virtual-switch[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in IBM Documentation.
 
 * See link:http://public.dhe.ibm.com/software/dw/linux390/perf/zvm_hpav00.pdf[Scaling HyperPAV alias devices on Linux guests on z/VM] for performance optimization.
```

Comment:
This link points to a site that cannot be reached. Update this link also? 

![Screenshot from 2023-07-31 11-40-29](https://github.com/openshift/openshift-docs/assets/57954076/6aad94cc-b8fa-431d-9d22-89949c997870)

Reply:
Hi @mburke5678 . Would you be OK to verify my peer review comments?

Reply:
@dfitzmau Good catch! The publication server recently changed from http to https.

Reply:
Thanks, @SNiemann15 . The link now works! Just need @mburke5678 to approve my peer review.